### PR TITLE
build with boost 1.65

### PR DESCRIPTION
**Pull request by randy3k**
I have just successfully built rstudio with boost 1.65 rather than 1.63. Two necessary changes are needed,

1. the `include <iostream>` is needed for `std:cerr`. May be because `boost/utility.hpp` doesn't include `iostream` now!?

2. instead of `boost::posix_time::time_duration::sec_type`, use `boost::uint64_t` to avoid ambiguous conversion in boost 1.65.

### CODE REVIEW

File: src/cpp/session/modules/SessionSVN.cpp [hunk 1]
```diff
@@ -1190,7 +1190,7 @@ struct CommitInfo
    std::string author;
    std::string subject;
    std::string description;
-   boost::posix_time::time_duration::sec_type date;
+   boost::uint64_t date;
```
Comment (kevinushey) [inline src/cpp/session/modules/SessionSVN.cpp]: Why was this change necessary? (what compile error were you seeing with Boost 1.65?)

FWIW it looks like `sec_type` changed from `int32_t` in Boost 1.63.0 to `std::time_t` in Boost 1.65.0.

Comment (randy3k) [reply src/cpp/session/modules/SessionSVN.cpp]: I got an ambiguous convention error from `sec_type` to `value_type` in [this line](https://github.com/randy3k/rstudio/blob/90701c01101e96933729f57f2f39364d60a31162/src/cpp/session/modules/SessionSVN.cpp#L1436), likely that the json array `pDates ` [doens't understand `sec_type`].

**Comment by jmcphers** [comment]
@randy3k appreciate the contribution!
